stacks-signer/src/v0/signer.rs

```diff
@@ -356,6 +356,7 @@ impl Signer {
         {
             // Block is too old. Drop it with a warning. Don't even bother broadcasting to the node.
             warn!("{self}: Received a block proposal that is more than {} secs old. Ignoring...", self.block_proposal_max_age_secs;
+                "signer_signature_hash" => %block_proposal.block.header.signer_signature_hash(),
```

Comment:
Nit: all of the other structured logs in this file call it `signer_sighash`. It's probably good for debugging if they all use the same name.

Reply:
Done!